cmake/Modules/FindComputeCpp.cmake

```diff
@@ -50,8 +50,8 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
       message(STATUS "host compiler - clang ${CMAKE_CXX_COMPILER_VERSION}")
     endif()
 else()
-  message(WARNING
-    "host compiler - Not found! (ComputeCpp supports GCC and Clang, see readme)")
+  set (COMPUTECPP_PLATFORM_SPECIFIC_ARGS "-fno-ms-compatibility")
```

Comment:
Is this a clang-only flag? Would it make sure that MSVC-isms don't sneak into the SDK?

Reply:
This is a workaround for a Clang bug there is no fix to just yet.

http://clang-developers.42468.n3.nabble.com/Why-in-C-11-target-win32-char16-t-disappears-td4034906.html

The problem is, that strangely Clang when put into C++11/14 mode does not define the fundamental types: char16_t and char32_t which plague the STL all over. If no-ms-compat is passed onto the compiler, it does. Hence when MSVC hosts Clangs STL, it is required to put into strict C++ mode; this flag turns off some MSVC weirdness. Luckily, MSVC is becoming conforming enough that the STL compiles fine in this mode.

Reply:
Bizarre! Fair enough, whatever makes it work.